inference/conflict.go

```diff
@@ -97,7 +97,8 @@ func (n *nilFlow) String() string {
 	lastNonnilNodeIndex := len(n.nonnilPath) - 1
 	n.nonnilPath[lastNonnilNodeIndex].reason = fmt.Sprintf("%s (must be NONNIL)", n.nonnilPath[lastNonnilNodeIndex].reason)
 
-	return pathString(n.nilPath) + pathString(n.nonnilPath)
+	flow := pathString(n.nilPath) + pathString(n.nonnilPath)
+	return flow
```

Comment:
Why this change?

Reply:
It seems to be stemming from a rebasing issue.